ASTProcessor/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+	application
+	id("org.springframework.boot") version "3.3.4"
```

Comment:
please create a new file to reference the version of the libraries, please do so for all the libraries on this file

Reply:
I didn't understand very well, could you give me a little more information or could you explain it to me in another way please?

Reply:
It was corrected, you can see all the versions of the libraries within the `gradle/libs.versions.toml` file that is in the Argos project.